mycroft/skills/intent_service.py

```diff
@@ -285,6 +301,9 @@ def handle_utterance(self, message):
             except Exception as e:
                 LOG.exception(f"Failed to set lingua_franca default lang to {lang}")
 
+            # Get utterance utterance_plugins additional context
+            message = self._get_transformers_context(message, lang)
+
             utterances = message.data.get('utterances', [])
             combined = _normalize_all_utterances(utterances)
```

Comment:
Should normalization move to a default plugin? (maybe a TODO for later)

Reply:
left a TODO comment, a plugin seems appropriate, but in general i don''t think we want this step here, it was meant to make life easier for adapt but feels like it belongs in the intent parsers, will revisit once I get back into intents #100